src/gcc.rs

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright 2021 IBM Corp.
+
+use crate::collector::*;
+use regex::Regex;
+
+
+pub fn version(col: &mut Collector) -> Result<CollectorValue, CollectorErr> {
+    let re = Regex::new(r"gcc version \d.*").unwrap();
+
+    println!("Collecting gcc version...");
+
+    let out = col.run_command("gcc -v");
+    
+    let output = String::from_utf8(out.stderr).unwrap();
+    
+    let caps = re.captures(&output).unwrap();
```

Comment:
Perhaps use `?` here in the event of a parse failure, collectors should return an error for non-programmatic errors.